src/DynamoRevitIcons/DynamoRevitIcons.csproj

```diff
@@ -34,7 +34,9 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="DSRevitNodesUIImages.resx" />
-    <EmbeddedResource Include="RevitNodesImages.resx" />
+    <EmbeddedResource Include="RevitNodesImages.resx">
```

Comment:
Oh so this was originally in the other PR https://github.com/DynamoDS/DynamoRevit/pull/2910/files#diff-42a24daedb6c64bb3a1c76755ee376806e5502d21e4a904a55360fe74f876a91R37 here. Is that the only change?